examples/HF_example_usage.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+from huggingface_connector import hf_atlasdataset
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Create an AtlasDataset from a Hugging Face dataset.')
+    parser.add_argument('--dataset_identifier', type=str, required=True, help='The Hugging Face dataset identifier')
```

Comment:
you may want to explicitly specify in the arg that it's a hf_dataset_identifier because atlas has its own datasets